src/styles/components/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  transition-duration: 0.3s;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    position: relative;
+    padding: 23px 0;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), vertical padding and margins should be set using `em` units instead of `px` to ensure better scalability and maintainability of the styles.